Description: migrate to fmodex
 Migrate from fmod to fmodex
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Last-Update: 2020-04-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Flawed/Sources/Music/Music.hpp
+++ b/Flawed/Sources/Music/Music.hpp
@@ -1,7 +1,7 @@
 #ifndef __MUSIC_HPP__
 #define __MUSIC_HPP__
 
-#include <FMOD/fmod.hpp>
+#include <fmodex/fmod.hpp>
 
 #include <string>
 
--- a/Flawed/Sources/Music/Music.cpp
+++ b/Flawed/Sources/Music/Music.cpp
@@ -1,7 +1,7 @@
 #include "Music.hpp"
 
 #include <iostream>
-#include <FMOD/fmod_errors.h>
+#include <fmodex/fmod_errors.h>
 
 Music :: Music(void)
 {
